modules/apps/product-navigation/product-navigation-applications-menu-web/src/main/resources/META-INF/resources/js/ApplicationsMenu.js

```diff
@@ -26,6 +26,17 @@ import {fetch, navigate, openSelectionModal} from 'frontend-js-web';
 import PropTypes from 'prop-types';
 import React, {useRef, useState} from 'react';
 
+const OPEN_MENU_TITLE_TPL = `
```

Comment:
When investigating about Tooltips on DXP I noticed nobody is instancing a ClayTooltip manually on module context. They are just using our tooltip-support utility and I decided to use it. However passing HTML to the tooltip content can be very painful with these templates.

Reply:
Would be awesome if We had a hook or something to pass a JSX to the tooltip and control it(showing and hiding).

Reply:
Now I understand why you were having those alignment issues. I think if you change this to:
```
const OPEN_MENU_TITLE_TPL = `<div>${Liferay.Language.get('open-menu')}</div><kbd class="c-kbd c-kbd-dark"><kbd class="c-kbd">⌘</kbd><span class="c-kbd-separator">+</span><kbd class="c-kbd">M</kbd></kbd>`;
```
it should render correctly.

Reply:
Now is rendering correctly. Thanks Pat!

Reply:
I noticed the markup didn't change since our first try. Why inlining the whole markup string make it work? 🤔

Reply:
It removes all the new lines and tabs. `white-space: pre-line` and `white-space: pre` makes the browser render new lines and spaces.